### PR TITLE
pull request

### PR DESCRIPTION
**Pull request by adamclasic**
## Design Teardown

This project consists of building a heatmap of the Smashing magazine website.

-Added a header heat map
-Added article content
-Added smashing tv section heat map
-Added a footer

[Live Version](https://adamclasic.github.io/Design-Teardown)

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @adamclasic and @Morrism1 

### Good job so far :clap: However, there are a few changes you need to work on before your project could be approved.
- Remove the horizontal scroll bar in your project
- Remove commented out code in your css file

- Once you make these changes resubmit for another review. I wish you all the best.
- HAppy coding!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+          porro perferendis cumque! Assumenda sed ad quibusdam quo fugiat
+          vitae magni.
+        </p>
+      </article>
+
+<<<<<<< HEAD
+      <div class="articles">
+      </div>
+      <div class="articles">
+=======
```
Comment (euqueme) [inline index.html]: Remove lines #65 to #69 as this seem to be a GitHub merge correction

File: index.html [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+      <article class="articles">
+        <h2> AD</h2>
+      </article>
+      <article class="articles">
+>>>>>>> 59f37dfc0c2bdafb473455ea148887bf70b6d606
```
Comment (euqueme) [inline index.html]: remove line #74

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @adamclasic and @Morrism1  :balloon:  :lollipop: 

### Thank you for doing the changes by the previous reviewer, your project looks good :ok_hand: 

## There is a couple of follow-up changes to make but there were some requirements in the Odin project that you missed and I can't request them as mandatory since they weren't pointed out before but please consider address them :point_down: 

## Follow-up

- [ ] remove the merge lines added by Github, they are making your project behave strangely (I have commented your code bellow :arrow_double_down:  something shifted? let's fix it!)

- [ ] Fix the errors detected by the [HTML validator](https://validator.w3.org/#validate_by_upload)

image

## Playing with Typography

- [ ] [OPTIONAL] Please add at least 2 fonts from Google fonts, that look like the Smashing magazine page fonts, and use them in your articles instead of the ones that you are currently using

image

## Footer

- [ ] [OPTIONAL] the footer should contain links (`<a>` tags) like the original page

image

## General

- [ ] [OPTIONAL] I encourage the use of  the `<nav>` tag for the major navigation links

### that's it! :raised_hands: 

### please :pray: make these changes :memo:  and submit back for a code review when you are done :rocket: 

Happy coding! :woman_technologist: 

Any question?  reach to me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Review by mosesogwo — APPROVED** [review]
**Hi, your project is approved :heavy_check_mark:**

You can now merge to master.

**Cheers:stars:**

**Pull request merged**